presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -109,6 +110,14 @@ public static Slice fromHexVarchar(@SqlType(StandardTypes.VARCHAR) Slice slice)
         return Slices.wrappedBuffer(result);
     }
 
+    @Description("return the md5")
+    @ScalarFunction("md5")
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice md5(@SqlType(StandardTypes.VARCHAR) Slice slice)
```

Comment:
This function should operate on `VARBINARY`. We should discuss if we want a `VARCHAR` version, and if we do, it needs to be documented as "compute md5 hash of UTF-8 encoded string"